### ISSUE TITLE
Error: SQLITE_ERROR: duplicate column name

### ISSUE THREAD

**Issue by ibbatta** [social-manager-tools/instagram-bot.js#125]
- **Bot Version:** v0.9.5
- **Node Version:** v8.9 (Bot require >= 7.6)
- **Operating System:** UbuntuMATE 16.04
- **Browser:** Google Chrome 69

### Expected Behavior

Please describe the program's expected behavior.

```
I expect the normal "like behavior" for the bot
```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```
When I start bot it return these errors:
2018-10-08 09:32:01 [WARNING] init_db users ADD COLUMN hashtag: Error: SQLITE_ERROR: duplicate column name: hashtag
2018-10-08 09:32:01 [WARNING] init_db users ADD COLUMN inserted_at: Error: SQLITE_ERROR: duplicate column name: inserted_at
```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. run the bot

### Screenshots (Optional)

If the error is graphical in nature it is helpful to provide a screenshot. 

![screenshot at 2018-10-08 11 45 03](https://user-images.githubusercontent.com/4375285/46602161-afc94d80-caef-11e8-9880-3ee996970187.png)
![screenshot at 2018-10-08 11 45 23](https://user-images.githubusercontent.com/4375285/46602162-afc94d80-caef-11e8-966c-a844ed2035d2.png)

**Comment by ptkdev** [comment]
This is warning for me but bot work correctly.

**Comment by ibbatta** [comment]
After some co-op tests I found that bot works only in debug mode with command line launcher.

**Comment by ptkdev** [comment]
Fixed in **`v0.9.8`**

<a href="http://patreon.ptkdev.io" alt="Patreon Backer"><img src="https://img.shields.io/badge/donate-patreon-F87668.svg"></a> <a href="http://paypal.ptkdev.io" alt="Paypale Donate"><img src="https://img.shields.io/badge/donate-paypal-46AFE0.svg"></a>  <a href="http://coffee.ptkdev.io" alt="Ko-fi Donate"><img src="https://img.shields.io/badge/buy%20me-coffee-4B788C.svg"></a>
